.github/workflows/release.yml

```diff
@@ -23,27 +20,27 @@ jobs:
 
       - name: Setup git env
         run: |
-          git config --global user.email "ckipp@pm.me"
-          git config --global user.name "coc-metals CI"
+          git config --global user.email "bot@scalameta.org"
```

Comment:
I know we aren't technically using the scalameta bot, but I wanted this to be consistent with the vscode extension.